runtime/caml/alloc.h

```diff
@@ -52,27 +67,12 @@ CAMLextern value caml_alloc_sprintf(const char * format, ...)
 CAMLextern value caml_alloc_some(value);
 
 typedef void (*final_fun)(value);
-CAMLextern value caml_alloc_final (mlsize_t wosize,
+CAMLextern value caml_alloc_final (mlsize_t, /*size in words*/
                                    final_fun, /*finalization function*/
                                    mlsize_t, /*resources consumed*/
                                    mlsize_t  /*max resources*/);
 
-CAMLextern int caml_convert_flag_list (value, int *);
-
-/* Convenience functions to deal with unboxable types. */
-Caml_inline value caml_alloc_unboxed (value arg) { return arg; }
-Caml_inline value caml_alloc_boxed (value arg) {
-  value result = caml_alloc_small (1, 0);
-  Field (result, 0) = arg;
-  return result;
-}
-Caml_inline value caml_field_unboxed (value arg) { return arg; }
-Caml_inline value caml_field_boxed (value arg) { return Field (arg, 0); }
-
-/* Unannotated unboxable types are boxed by default. (may change in the
-   future) */
-#define caml_alloc_unboxable caml_alloc_boxed
-#define caml_field_unboxable caml_field_boxed
+CAMLextern int caml_convert_flag_list (value, const int *);
```

Comment:
The removal of `caml_alloc_unboxed`/`caml_alloc_boxed` looks like a bad merge.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/40a2c50f73eb6da5d0d34959187bdb184ab6155b